docs/docusaurus/versioned_docs/version-1.1.0/lte/upgrade.md

```diff
@@ -25,9 +25,11 @@ software version in this tab has changed. If it hasn't changed, something
 probably went wrong with the autoupgrade. You will probably have to SSH into
 the AGW to troubleshoot the installation (see the next section).
 
-If you want to roll out an upgrade slowly to your fleet, you can segment your
-AGWs into different tiers and upgrade tiers one-by-one. Use the NMS to create
-new tiers and assign your AGWs to them.
+It is recommended that you conduct upgrades on a test prior to rolling it out to your entire network. 
+
```

Comment:
This is good documentation. However it will be really useful to do it with screenshot or a gif. Text description is very hard to follow.